install.php

```diff
@@ -124,7 +124,10 @@
                     { code: 'mbstringLibrary', label: 'Mbstring PHP Extension is required' },
                     { code: 'sslLibrary', label: 'OpenSSL PHP Extension is required' },
                     { code: 'zipLibrary', label: 'ZipArchive PHP Library is required' },
-                    { code: 'gdLibrary', label: 'GD PHP Library is required' }
+                    { code: 'gdLibrary', label: 'GD PHP Library is required' },
+                    { code: 'mod_rewrite', label: 'PHP mod_rewrite module is required' },
```

Comment:
OctoberCMS doesn't actually require mode_rewrite on non-Apache servers, you'd need to add an additional check to see if this is an Apache server.

Reply:
True, I changed that and something more.
And regarding json extension check, and before you say "As of PHP 5.2.0, the JSON extension is bundled and compiled into PHP by default.", I managed to find a PHP 5.6 misconfigured server without JSON enabled...